src/components/BaseLineChart.vue

```diff
@@ -36,26 +36,27 @@ export default {
       required: true,
     },
   },
+  created() {
+    this.instanceId = this.id || this._uid;
```

Comment:
Where does `this._uid` come from?

Reply:
It's the internal Vue component ID, so it'll be unique. :) might be worth verifying multiple instances are unique though, which they should be.

Reply:
It looks like it's not officially supported by Vue and considered an unstable feature:  https://github.com/vuejs/vue/issues/5886

Maybe we should use crypto or another lib to generate it?

Reply:
Fine with me, I'm also fine not using it and requiring the id passed as a prop. Just need safety that it'll always be unique.

Reply:
You also have the benefit of no server side component at the moment, but it would be worth considering it since for non game pages we might want to move there eventually for SEO and caching reasons.

Reply:
Yeah I think making it a required prop could be simpler and skip the need for this created hook and auto generating a default. @jerryh-jr let's nix the `created()` hook and make `id` a `required` prop.